test/e2e/simple.sh

```diff
@@ -47,5 +47,18 @@ RELEASE_IMAGE_LATEST="$( curl -s -H "Accept: application/json"  "https://api.ope
 export RELEASE_IMAGE_LATEST
 os::cmd::expect_success "ci-operator --secret-dir ${PULL_SECRET_DIR} --target [release:latest] --config ${suite_dir}/dynamic-releases.yaml"
 unset RELEASE_IMAGE_LATEST
+os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "e2e/simple/optional-operator"
+if [[ -z "${PULL_BASE_SHA:-}" ]]; then
```

Comment:
Confused about these, and why `$JOB_SPEC` doesn't cover it?

Reply:
It should. Changed to use that instead.

Reply:
Did you?